app/lib/hosting_environment.rb

```diff
@@ -21,9 +21,13 @@ def self.hostname
 
   def self.phase
     if production?
-      'beta'
+      'Beta'
+    elsif sandbox_mode?
+      'Sandbox'
```

Comment:
This should stay as is was. We sometimes put other environments in sandbox _mode_, and it shouldn't be confused with the sandbox environment.

Reply:
Reverted this change. I wonder… will capitalising `environment_name.capitalize` mean `sandbox` appears capitalised?

Reply:
Yes it will!